app/Config/App.php

```diff
@@ -239,13 +239,15 @@ class App extends BaseConfig
 	| CSRFTokenName   = The token name
 	| CSRFHeaderName  = The header name
 	| CSRFCookieName  = The cookie name
+	| CSRFCookieSameSite  = The cookie SameSite attribute (none, Lax or Strict)
```

Comment:
Case for these should match what a developer is expected to use. You're using all lowercase below, but these are listed out as Initial Caps.